### PR TITLE
add dark mode

### PR DESCRIPTION
**Pull request by khareyash05**
With reference to issue #4 
Added Dark Mode

### CODE REVIEW

**Comment by sagar-barapatre** [comment]
Provide us with some screenshots or GIF's

File: src/App.js [hunk 1]
```diff
@@ -6,30 +6,39 @@ import ChatBotComponent from './components/chatbot';
 import './App.css';
 import Board from './board.jpg';
 import Footer from './components/Footer';
+import { ThemeProvider } from 'styled-components';
+import { useDarkMode } from './components/useDarkMode';
+import Toggle from './components/Toggle';
+import { lightTheme, darkTheme } from './components/themes';
```
Comment (sagar-barapatre) [inline src/App.js:12]: it should be ./component/theme

**Review by sagar-barapatre — COMMENTED** [review]
I tested your changes in the my local setup, no toggle of theme is taking place. I mean nothing is changing on the toggling the button

File: src/App.js (hunk 1 above)
Comment (khareyash05) [reply src/App.js:12]: @sagar-barapatre I am not able to extract I have no file or folder named theme Its just themes.js Also the folder is named component right?

Comment (khareyash05) [reply src/App.js]: @sagar-barapatre  I have changed file locations kindly see if this worked

**Comment by sagar-barapatre** [comment]
Provide some screenshots or GIF's for reference...so that whatever I test and whatever you show via SS matches.

**Pull request closed without merging**